tests/helm_template_generator.py

```diff
@@ -93,6 +95,8 @@ def render_chart(
             "--values",
             tmp_file.name,
         ]
+        if namespace:
+            command.extend(["--namespace", namespace])
         if show_only:
```

Comment:
set show_only default to `()` and delete this line

Reply:
Wouldn't setting show_only to `()` trigger the same empty data in assignment bug that you brought up earlier?

Reply:
tuples are immutable, so it wouldn't. When we use `[]`, python stores a reference to the list's location as the default variable. When we use `()`, python will literally store an empty tuple.